<launch>
  <node pkg="complete_mission" type="check_camera" name="" output="screen">
      <rosparam command="load" file="$(find complete_mission)/config/gjs.yaml" />
  </node>
</launch>
